### ISSUE TITLE
Don't let me create envs with not allowed names

### ISSUE THREAD

**Issue by amercader** [boxkite/datacats#66]
So I was really excited about deploying my little env named `rtl` when:

```
datacats deploy
Please choose a name at least 4 characters long
and containing only lowercase letters and numbers
```
Can I be warned in advance? :)

Also just wondering is there are reason for not supporting underscores / hyphens?

**Comment by deniszgonjanin** [comment]
Sorry @amercader, we should have a warning somewhere in there. Ideally datacats should warn you about proper names when you create a project on the command line, but we didn't want to tie the open source project too closely to the (currently) closed server-side implementation on datacats.com. In the future our hope is that you could use the datacats project to deploy easily to any cloud provider, which may not have the same naming restrictions as datacats.com, so it doesn't make sense to hardcode the naming limitations in the project.

If you really want "rtl", I can take a look at making that happen. As for underscores/hyphens, that was also just a stylistic choice, like the 4 character minimum.

**Comment by amercader** [comment]
@deniszgonjanin makes sense. Maybe in the future the cloud provider can provide an endpoint to validate the names.

**Comment by wardi** [comment]
You can specify the name you would like to deploy to (the "target name") too: http://docs.datacats.com/commands.html#deploy

**Comment by wardi** [comment]
Fixed by f4de559
